css/_videolayout_default.scss

```diff
@@ -95,8 +95,8 @@
         }
 
         .connection-indicator,
-        div.indicator-container,
-         {
+        div.indicator-container
+        {
```

Comment:
While you're being a good citizen and fixing the trailing comma, the left bracket should be moved up a line to 98.